src/essreflectometry/amor/calibrations.py

```diff
@@ -61,23 +57,17 @@ def calibration_factor(
     data_array:
         Data array to get q-bins/values from.
     m_value:
-        m-value for the supermirror. Defaults to 5.
+        m-value for the supermirror.
     critical_edge:
-        Supermirror critical edge. Defaults to 0.022 1/angstrom.
+        Supermirror critical edge.
     alpha:
-        Supermirror alpha value. Defaults to 0.25 / 0.088 angstrom.
+        Supermirror alpha value.
 
     Returns
     -------
     :
         Calibration factor at the midpoint of each Q-bin.
     """
-    if m_value is None:
-        m_value = sc.scalar(5, unit=sc.units.dimensionless)
-    if critical_edge is None:
-        critical_edge = 0.022 * sc.Unit('1/angstrom')
-    if alpha is None:
-        alpha = sc.scalar(0.25 / 0.088, unit=sc.units.angstrom)
     q = data_array.coords['Q']
     if data_array.coords.is_edges('Q'):
         q = sc.midpoints(q)
```

Comment:
Can we remove the `data_array` input and instead use `QBins` directly? It does not look like we actually need the data?